pom.xml

```diff
@@ -163,6 +164,16 @@
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-javadoc-plugin</artifactId>
 				</plugin>
+				<plugin>
```

Comment:
Why do we need the extra dependencies here? Does the outputTimestamp thing need these?

Reply:
@parrt
it's not really new dependencies: it's an update of plugin versions for plugins that are already used (default Maven lifecycle for jars), to use plugins that support Reproducible Builds (use the outputTimestamp)